user/src/components/FormList/FormList.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import Button from '@/components/Button';
+import FormContainer from '@/components/FormContainer';
+import { FormItem } from './type';
+
+type FormListProps = {
+  items: FormItem[];
+};
+
+const FormList: FC<FormListProps> = ({ items }) => {
+  return (
+    <FormContainer>
+      {items.map((item, index) => (
+        <div key={index} className="w-auto md:w-72 flex flex-col">
```

Comment:
Figmaのデザインに沿うと、PC表示時には内部要素を400px取っているようでした。
| ![image](https://github.com/user-attachments/assets/f2ccea46-14f0-406d-8e18-18d1a992f656) | ![image](https://github.com/user-attachments/assets/edfdcd46-c674-4e2c-942f-a2e8c1e073a4) |
| -- | -- |


```suggestion
        <div key={index} className="w-auto md:w-[400px] flex flex-col">
```